Fixed & updated Unity docs for road to 0.1.0 (#3603)
getsentry/sentry-docs: src/includes/getting-started-config/unity.mdx

> The minimum configuration required is the [DSN](/product/sentry-basics/dsn-explainer/) to your project.
> You can find the DSN in your project settings on [sentry.io](https://sentry.io/). If you are logged in you can also select your project and copy its DSN directly from here:

Reviewer: No need to explain the DSN on this page (as we have a link to an explanation of it). 
```suggestion
The minimum configuration required is the [DSN](/product/sentry-basics/dsn-explainer/) for your project.
```

Author: I removed the DSN explainer but moved the "get it from here"-part into a note below because if you're not logged in it just looks like a fancy code-snippet.

```

> ___PUBLIC_DSN___

Reviewer: delete
